packages/core/src/ckb-cli.ts

```diff
@@ -1,21 +1,165 @@
 import { execSync } from 'node:child_process'
+import { join } from 'node:path'
+import { writeFileSync, rmSync } from 'node:fs'
+import type { ExecSyncOptions } from 'node:child_process'
+import { key } from '@ckb-lumos/hd'
+import { CKBCLIDownloader } from './download'
+import { KuaiError } from '@ckb-js/kuai-common'
+import { ERRORS } from './errors-list'
+import { DEFAULT_NETWORKDS, DEFAULT_CKB_CLI_VERSION, DEFAULT_DEVNET_ACCOUNT_PKS } from './constants'
+import { AccountCommands, CKBCLIAccount, UtilCommands, TxCommands } from './type/ckb-cli'
+import { kebabCaseKeys } from './util/case'
+import { extractJsonFromString } from './util/json'
+import { omit } from 'lodash'
 
-interface CkbCliAccount {
-  address: {
-    mainnet: string
-    testnet: string
+interface CKBCLIOptions {
+  url?: string
+  outputFormat?: 'json' | 'yaml'
+}
+
+export class CKBCLI {
+  binPath = 'ckb-cli'
+  homeDir = '~/.ckb-cli'
```

Comment:
Windows may have no directory named `~`

Reply:
I use the `env-paths` to store the dev scripts. Could we use here?
https://github.com/ckb-js/kuai/blob/c2845169de81817fd2fd397032672f79bf73aebd/packages/core/src/constants.ts#L34

https://github.com/ckb-js/kuai/blob/c2845169de81817fd2fd397032672f79bf73aebd/packages/core/src/helper.ts#L32